tests/BenchmarkDotNet.Tests/Exporters/ApprovalTests.cs

```diff
@@ -1,4 +1,4 @@
-﻿#if CLASSIC || NETCOREAPP2_0
+﻿#if CLASSIC
```

Comment:
Probably it's a merge bug. We need `#if CLASSIC || NETCOREAPP2_0` here (now we can run ApprovalTests on Linux and macOS on .NET Core 2.0)